src/python/pants/backend/codegen/protobuf/protoc.py

```diff
@@ -19,7 +19,9 @@ class Protoc(TemplatedExternalTool):
         "24.4|macos_x86_64|6c3b6bf4038d733b6d31f1cc4516a656570b5b5aafb966b650f8182afd0b98cf|2121366",
         "3.20.1|linux_arm64 |8a5a51876259f934cd2acc2bc59dba0e9a51bd631a5c37a4b9081d6e4dbc7591|1804837",
         "3.20.1|linux_x86_64|3a0e900f9556fbcac4c3a913a00d07680f0fdf6b990a341462d822247b265562|1714731",
-        # 3.20.1 does not support macos_arm64. See https://github.com/protocolbuffers/protobuf/issues/9893
+        # 3.20.1|macos_arm64 below is an x86 binary. Needs Rosetta to run on Apple Silicon.
+        # see https://github.com/protocolbuffers/protobuf/issues/9893
+        "3.20.1|macos_arm64 |b362acae78542872bb6aac8dba73aaf0dc6e94991b8b0a065d6c3e703fec2a8b|2708249"
```

Comment:
This hash is different to the x86-64 one below. Just confirming that's expected?

Reply:
Yes I did a double-take on that too, but it was different before. See: https://github.com/pantsbuild/pants/pull/20058/files 

Just rechecked it:
```
$ shasum -a 256 protoc-3.20.1-osx-aarch_64.zip
b362acae78542872bb6aac8dba73aaf0dc6e94991b8b0a065d6c3e703fec2a8b  protoc-3.20.1-osx-aarch_64.zip
```